modules/core/02-client/types/msgs_test.go

```diff
@@ -1,17 +1,23 @@
 package types_test
 
 import (
+	"errors"
 	"testing"
 	"time"
 
 	"github.com/golang/protobuf/proto" //nolint:staticcheck
 	"github.com/stretchr/testify/require"
 	testifysuite "github.com/stretchr/testify/suite"
 
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
```

Comment:
ditto on sdkerrors, I see `ibcerrors` imported also below but unsure why both are used together and if its okay or not

Reply:
you can see an example of this in the `e2e/tests/transfer/authz_test`, for the same reason as i stated above (the err return in this case is from a version containing the feature, which would then return an ibcerror instead of the sdkerror from a previous version)